Extension/src/LanguageServer/extension.ts

```diff
@@ -85,6 +165,32 @@ export function activate(activationEventOccurred: boolean): void {
         }
     });
 
+    vcpkgDbPromise = initVcpkgDatabase();
```

Comment:
I don't believe this belongs here.  You should only read the vcpkg database in the `realActivation` function.